From 225c150420f4c4e97b271321eb2ad15145a070ba Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 30 Oct 2018 15:05:46 +0000
Subject: [PATCH 0853/1000] MANTIS 0049200 Illumination device is shown as
 hardware type Scanner

platform.idl
V3.9
added LC_INDICATOR for illumination device
added LC_AUDIO for headset device

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1392 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index 7c09e2ad..b52e6b26 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "3.8";
+    const string interface_version = "3.9";
 
     /** Data type for text lists. */
     typedef sequence<string> list;
@@ -201,7 +201,9 @@ module PlatformManagement
       LC_DISK,
       LC_NETWORK,
       LC_UNDEFINED,
-      LC_PLF
+      LC_PLF,
+      LC_INDICATOR,
+      LC_AUDIO
     };
     
     struct LoggingComponentData 
-- 
2.41.0.windows.1

